app/src/main/java/com/antonioleiva/bandhookkotlin/ui/screens/album/AlbumLayout.kt

```diff
@@ -30,7 +35,7 @@ class AlbumLayout : ActivityAnkoComponent<AlbumActivity> {
                 backgroundResource = android.R.color.darker_gray
             }.lparams(width = matchParent)
 
-            toolbar = toolbar(R.style.ThemeOverlay_AppCompat_Dark_ActionBar)
+            toolbar = themedToolbar(R.style.ThemeOverlay_AppCompat_Dark_ActionBar)
```

Comment:
Anko 0.10 has renamed all themed constructors to `themeXView`. Original constructors are no longer present, so `themed` prefix has been added where needed.